### ISSUE TITLE
Make jump trajectiles match custom formation dots.

### ISSUE THREAD

**Issue by Skasi** [ZeroK-RTS/Zero-K#421]
Using custom formation jump is possible by selecting multiple units, then Jump and drawing a line, but the GUI does not support displaying this.

When having multiple units selected and drawing a jump-line command the jump arcs don't spread up along the line properly.
![screen00469](https://cloud.githubusercontent.com/assets/132906/5331102/840226c6-7e19-11e4-9e9e-384097e45503.png)

**Comment by Orfelius** [comment]
Can this be closed since jump line command is a thing now?

**Comment by sprunk** [comment]
No, it's about the curved trajectory display.

**Comment by sprunk** [comment]
Dots only depend on the customformation shape, but the jump trajectiles also depend on which unit gets assigned to which dot which is more expensive to calculate and can change if the jumpers move around.

**Comment by GoogleFrog** [comment]
Agreed. Perhaps check if BAR custom formations has some algorithm to assign units to dots.
